project/crud.py

```
@@ -31,9 +39,36 @@ class TaggedResult(BaseModel):
+        return get_postgres_db(settings=get_settings())
+
+    def test_connection(self):
+        """Tests connection and binding of tables needed for event logging."""
+        with self.db:
+            pass
+
+    def setup(self):
+        """Initializes a configured database with the help of the database proxy that is already bound to the models."""
+        logger.info("Initializing connection to Postgres for storing tags and result metadata.")
+        if proxy.obj is not None:
+            raise pw.PeeweeException("Database proxy is already initialized.")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)